torch_geometric/data/data.py

```diff
@@ -308,6 +308,25 @@ def up_to(self, end_time: Union[float, int]) -> Self:
             store.up_to(end_time)
         return out
 
+    def train_val_test_split_time(
```

Comment:
Any objection to introduce a `TemporalSplit` as a `transform`?

Reply:
No objections :wink: Code updated.